infrastructure/lib/addons/fluxv2.ts

```diff
@@ -128,34 +127,6 @@ export class FluxV2 extends Construct {
   });
   kustomizationManifest.node.addDependency(chart);
 
-  if (props.repoAddonPaths) {
```

Comment:
This gives customer the flexibility to install addons from multiple repositories that we are not  opinionated by us. Any reason why we want to remove this flexibility ?

Reply:
`repoAddonPaths` was introduced for perfdash. And this flag is no longer need to install perfdash.
`repoAddonPaths` is basically doing the same as `kubectl apply -f <repoAddonPaths>`. And IMO it's more intuitive to use `kubectl apply -f`. e.g. applying some yaml file is more trivial with `kubectl apply` than running `cdk deploy` which sometimes give strange errors :/

Reply:
IIRC this is for flux to apply automatically addons from this folder - https://github.com/awslabs/kubernetes-iteration-toolkit/tree/main/infrastructure/k8s-config/clusters/kit-infrastructure 
Not just for perfdash. 
Or am i missing something here ?